packages/storybook/cxl-ui/cxl-app-layout/layout=2c-l.stories.js

```diff
@@ -1,8 +1,9 @@
 import { html } from 'lit-html';
-import { withKnobs, boolean } from '@storybook/addon-knobs';
+import { withKnobs, boolean, number } from '@storybook/addon-knobs';
```

Comment:
Avoid `withKnobs`, all new stuff should be controls https://github.com/conversionxl/aybolit/issues/153